sql-plugin/src/main/301until310-nondb/scala/com/nvidia/spark/rapids/shims/v2/Spark30XShims.scala

```diff
@@ -361,6 +361,8 @@ abstract class Spark30XShims extends Spark301util320Shims with Logging {
     adaptivePlan.initialPlan
   }
 
+  override def isNegativeDecimalScaleSupportEnabled: Boolean = true
```

Comment:
This is still too generic. It needs to mention something about casting strings to neg scale decimal